index.js

```diff
@@ -9,8 +9,23 @@
 
 var cp = require('child_process');
 
+var program = {
+	darwin: {
+		name: 'pbcopy',
+		args: []
+	},
+	win32: {
+		name: 'clip',
+		args: [],
+	},
+	linux: {
+		name: 'xclip',
+		args: ['-selection', 'clipboard']
+	}
+};
+
 function toClipboard(args, cb) {
-  var proc = cp.spawn(program(), {
+  var proc = cp.spawn(program.name, program.args, {
```

Comment:
Seems like something got out of sync somehow. Its supposed to have `[process.platform]` right after the object definition.